ads/netletix.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript, loadScript, validateData} from '../3p/3p';
+import {startsWith} from '../src/string.js';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+/** @type {{nxasync:string}} */
```

Comment:
These `@type` declarations set here don't do anything and will be ignored by the Closure compiler 🙂 Please remove.